libs/api/domains/identity/src/lib/identity.module.ts

```diff
@@ -5,14 +5,15 @@ import { NationalRegistryXRoadModule } from '@island.is/api/domains/national-reg
 import { IdentityClientModule } from '@island.is/clients/identity'
 
 import { IdentityResolver } from './identity.resolver'
+import { IdentityLoader } from './identity.loader'
 
 @Module({
   imports: [
     NationalRegistryXRoadModule,
     CompanyRegistryModule,
     IdentityClientModule,
   ],
-  providers: [IdentityResolver],
+  providers: [IdentityResolver, IdentityLoader],
```

Comment:
Hm. Don't these need to be exported? Or how are they invoked elsewhere?

Reply:
Interesting, my theory is that because the DataLoaderInterceptor is loaded as global interceptor (doesn't matter if it is in the api app.module or in some domain module) it is able to resolve all data loaders. But I will add it in exports just for consistency and if the interceptor scopes ever changes.